python/requests/best-practice/use-response-json-shortcut.yaml

```diff
@@ -0,0 +1,17 @@
+rules:
+- id: python.requests.best-practice.use-response-json-shortcut
+  patterns:
+  - pattern-inside: import json; ...
+  - pattern-inside: import requests; ...
+  - pattern-inside: $RESP = requests.$METHOD(...); ...
+  - pattern: json.loads($RESP.text)
+  fix: $RESP.json()
```

Comment:
Will this fix keep the rest of the line? Not sure if `payload = ` will be kept, like in the test case example on line 10.

Reply:
it will keep it, just tested it: https://semgrep.dev/s/XBZ4/